### PR TITLE
Initial version

### PR DESCRIPTION
**Pull request by HernanRivasAcosta**
For issue #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # jem.js
 Just Erlang Maps for Javascript
+
+Jem.js encodes JavaScript objects in native erlang notation, after that, from Erlang you simply call ``erlang:binary_to_term(Binary)`` to obtain the same representation of a JSON you would usually get with jiffy.
+
+Why would you use this? Two reasons, first, processing lots of JSON objects could get expensive on the server side, and second, this object representation is more concise, so less bandwidth will be used. In other words, you can save money on the hosting.
```
Comment (HernanRivasAcosta) [inline README.md]: @Matias-Vera This could use your proofreading skills.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # jem.js
 Just Erlang Maps for Javascript
+
+Jem.js encodes JavaScript objects in native erlang notation, so that, from Erlang you can simply call ``erlang:binary_to_term(Binary)`` to obtain the same representation of a JSON you would usually get from jiffy.
+
+Why would you use this? Two reasons, first, processing lots of JSON objects could get expensive on the server side, and second, this object representation is more concise, so less bandwidth is required. In other words, you can save money on the hosting.
```
Comment (HernanRivasAcosta) [inline README.md:6]: @Matias-Vera This could use your proofreading skills.

**Comment by flaviogranero** [comment]
should I merge?

**Comment by elbrujohalcon** [comment]
@flaviogranero sure, if it's ok to you.

**Pull request merged**